src/store/messagesStore.js

```diff
@@ -542,6 +551,31 @@ const actions = {
 			return
 		}
 
+		if (message.parent && message.systemMessage === 'message_edited') {
```

Comment:
Maybe we include this code to the block above, to not duplicate lines?

Reply:
I thought about it and I put it separately for the sake of clarity. I will translate it there then.

Reply:
It's still there

Reply:
Don't forget to remove it